src/components/chip/chip.scss

```diff
@@ -6,15 +6,16 @@
   --calcite-chip-spacing-unit-s: theme("spacing.1");
   .image-container {
     @apply h-5 w-5;
+    padding-inline-start: theme("spacing[0.5]");
```

Comment:
new styles?

Reply:
Yeah, there were some new rules needed, mostly to achieve the proper spacing for icon/image-slot-only fixes from https://github.com/Esri/calcite-components/issues/4837, the `spacing[0.5]` itself already exists.